src/util/setupJest.ts

```diff
@@ -0,0 +1,6 @@
+import * as adapter from 'enzyme-adapter-react-16';
+import * as enzyme from 'enzyme';
+
+(global as any).fetch = jest.fn();
```

Comment:
I'd do this separately in the test that needs this, so we can reset the mock before each test run